db/schema.rb

```diff
@@ -201,11 +201,11 @@
     t.string   "name"
     t.string   "file_name"
     t.string   "run_type"
-    t.string   "state"
     t.integer  "accounting_plan_id"
     t.integer  "organization_id"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "state"
```

Comment:
I would prefer if we did not commit these kind of changes.
If it is ok by you, I can "revert" these db/schema.rb changes manually.